src/main/java/de/geolykt/enchantments_plus/util/ColUtil.java

```diff
@@ -31,1162 +31,1162 @@
-        case RED_SHULKER_BOX:
-            break;
-        case RED_STAINED_GLASS:
-            return AbstractDyeableType.GLASS;
-        case RED_STAINED_GLASS_PANE:
-            return AbstractDyeableType.GLASS_PANE;
-        case RED_TERRACOTTA:
-            return AbstractDyeableType.TERRACOTTA;
-        case RED_TULIP:
-            break;
-        case RED_WALL_BANNER:
-            return AbstractDyeableType.BANNER;
-        case RED_WOOL:
-            return AbstractDyeableType.WOOL;
-        case TERRACOTTA:
-            return AbstractDyeableType.TERRACOTTA;
-        case WHITE_BANNER:
-            return AbstractDyeableType.BANNER;
-        case WHITE_BED:
-            return AbstractDyeableType.BED;
-        case WHITE_CARPET:
-            return AbstractDyeableType.CARPET;
-        case WHITE_CONCRETE:
-            return AbstractDyeableType.CONCRETE;
-        case WHITE_CONCRETE_POWDER:
-            return AbstractDyeableType.CONCRETE_POWDER;
-        case WHITE_DYE:
-            break;
-        case WHITE_GLAZED_TERRACOTTA:
-            return AbstractDyeableType.GLAZED_TERRACOTTA;
-        case WHITE_SHULKER_BOX:
-            break;
-        case WHITE_STAINED_GLASS:
-            return AbstractDyeableType.GLASS;
-        case WHITE_STAINED_GLASS_PANE:
-            return AbstractDyeableType.GLASS_PANE;
-        case WHITE_TERRACOTTA:
-            return AbstractDyeableType.TERRACOTTA;
-        case WHITE_TULIP:
-            break;
-        case WHITE_WALL_BANNER:
-            return AbstractDyeableType.BANNER;
-        case WHITE_WOOL:
-            return AbstractDyeableType.WOOL;
-        case YELLOW_BANNER:
-            return AbstractDyeableType.BANNER;
-        case YELLOW_BED:
-            return AbstractDyeableType.BED;
-        case YELLOW_CARPET:
-            return AbstractDyeableType.CARPET;
-        case YELLOW_CONCRETE:
-            return AbstractDyeableType.CONCRETE;
-        case YELLOW_CONCRETE_POWDER:
-            return AbstractDyeableType.CONCRETE_POWDER;
-        case YELLOW_DYE:
-            break;
-        case YELLOW_GLAZED_TERRACOTTA:
-            return AbstractDyeableType.GLAZED_TERRACOTTA;
-        case YELLOW_SHULKER_BOX:
-            break;
-        case YELLOW_STAINED_GLASS:
-            return AbstractDyeableType.GLASS;
-        case YELLOW_STAINED_GLASS_PANE:
-            return AbstractDyeableType.GLASS_PANE;
-        case YELLOW_TERRACOTTA:
-            return AbstractDyeableType.TERRACOTTA;
-        case YELLOW_WALL_BANNER:
-            return AbstractDyeableType.BANNER;
-        case YELLOW_WOOL:
-            return AbstractDyeableType.WOOL;
-        default:
-            break;
+            case BLACK_BANNER:
+                return AbstractDyeableType.BANNER;
+            case BLACK_BED:
+                return AbstractDyeableType.BED;
+            case BLACK_CARPET:
+                return AbstractDyeableType.CARPET;
+            case BLACK_CONCRETE:
```

Comment:
yeah. You get the idea.
If you are submitting 2.2k lines of source code changes within a file, you should review if the changes made sense